discovery-provider/docker-compose.yml

```diff
@@ -511,6 +511,25 @@ services:
         max-buffer-size: 100m
       driver: json-file
 
+  mri-reporter:
```

Comment:
can we just call this mri so it stays consistent w/ the pedalboard name and docker tag?